packages/blocks/src/text-question/index.js

```diff
@@ -21,10 +17,7 @@ const TextQuestion = ( { attributes, className } ) => {
 
 	return (
 		<QuestionWrapper attributes={ attributes } className={ classes }>
-			<RichText.Content
-				tagName={ QuestionHeader }
-				value={ attributes.question }
-			/>
+			<QuestionHeader>{ attributes.question }</QuestionHeader>
```

Comment:
Do we need to use `RichText.Content`?
Using it seems to add a wrapper `<div />` which is affecting some styles.

Reply:
Unfortunately, yes. It doesn't matter with plain text but once you start styling it (bold, italic, etc.) or if you're using emojis it won't render them correctly otherwise :(.  
We actually were using plain `<div>`s on Crowdsignal forms at first and that's how we found out.

Since `QuestionHeader` is a styled component... I wonder if maybe turning it around would help?

```
<QuestionHeader as={ RichText.Content } value={ attributes.question } />
```

Reply:
Actually, just ran into that in #110, without RichText content HTML entities aren't getting decoded correctly.

Reply:
Got it! Thanks for clarifying. I'm reverting those changes.